src/Instagram/Transport/JsonTransportFeed.php

```diff
@@ -15,6 +16,8 @@ class JsonTransportFeed extends TransportFeed
      */
     private $endCursor;
 
+    private $requiresCookieKeys = ['rur', 'mid', 'mcd', 'urlgen', 'csrftoken'];
```

Comment:
`const REQUIRES_COOKIES_KEYS = [...]` is better here

Reply:
Thank you. I fixed the issue.  ↓ I used git rebase in practice.

```diff
--- a/src/Instagram/Transport/JsonTransportFeed.php
+++ b/src/Instagram/Transport/JsonTransportFeed.php
@@ -16,7 +16,7 @@ class JsonTransportFeed extends TransportFeed
      */
 
-    private $requiresCookieKeys = ['rur', 'mid', 'mcd', 'urlgen', 'csrftoken'];
+    const REQUIRES_COOKIES_KEYS = ['rur', 'mid', 'mcd', 'urlgen', 'csrftoken'];
 
     /**
      * JsonTransportFeed constructor.
@@ -87,15 +87,14 @@ class JsonTransportFeed extends TransportFeed
         // save to cache for next request
         $newCache = new Cache();
         $newCache->setUserId($cache->getUserId());
        if ($res->hasHeader('Set-Cookie')) {
            $saveCookies = [];
             foreach ($res->getHeaders()['Set-Cookie'] as $cookie) {
                 $setCookie = SetCookie::fromString($cookie);
-                if (in_array($setCookie->getName(), $this->requiresCookieKeys, true)) {
+                if (in_array($setCookie->getName(), self::REQUIRES_COOKIES_KEYS, true)) {
                     $saveCookies[] = $cookie;
```

Reply:
👍